docs/rest_api/endpoints/code_deployment/download_deployment_package_endpoint.yaml

```diff
@@ -1,6 +1,6 @@
 post:
   tags:
-    - Deployment package
+    - Code Deployment
```

Comment:
What do you think of renaming this to "Deployment" instead? Code doesn't really make much sense in the context of ovms.